src/main/resources/initialize.sql

```diff
@@ -0,0 +1,429 @@
+    "LastName" character varying(30) COLLATE pg_catalog."default" NOT NULL,
+    "Password" text COLLATE pg_catalog."default",
+    "IsDeleted" boolean NOT NULL,
+    CONSTRAINT "User_pkey" PRIMARY KEY ("UserID")
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE IF EXISTS public."User"
+    OWNER to postgres;
+
+
+--2 Table: public.OnlineDonor
+
+-- DROP TABLE IF EXISTS public."OnlineDonor";
+
+CREATE TABLE IF NOT EXISTS public."OnlineDonor"
+(
+    "OnlineDonorID" integer NOT NULL DEFAULT nextval('"OnlineDonor_OnlineDonorID_seq"'::regclass),
+    "NumberofDonations" smallint NOT NULL,
+    CONSTRAINT "OnlineDonor_pkey" PRIMARY KEY ("OnlineDonorID"),
+    CONSTRAINT "OnlineDonorForeignKey" FOREIGN KEY ("OnlineDonorID")
+        REFERENCES public."User" ("UserID") MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE IF EXISTS public."OnlineDonor"
+    OWNER to postgres;
+
+
+--3 Table: public.Admin
+
+-- DROP TABLE IF EXISTS public."Admin";
+
+CREATE TABLE IF NOT EXISTS public."Admin"
+(
+    "AdminID" integer NOT NULL DEFAULT nextval('"Admin_AdminID_seq"'::regclass),
+    "Status" boolean NOT NULL,
+    CONSTRAINT "Admin_pkey" PRIMARY KEY ("AdminID"),
+    CONSTRAINT "Admin_fkey" FOREIGN KEY ("AdminID")
+        REFERENCES public."User" ("UserID") MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE IF EXISTS public."Admin"
+    OWNER to postgres;
+
+
+--4 Table: public.Mentor
+
+-- DROP TABLE IF EXISTS public."Mentor";
+
+CREATE TABLE IF NOT EXISTS public."Mentor"
+(
+    "MentorID" integer NOT NULL DEFAULT nextval('"Mentor_MentorID_seq"'::regclass),
+    CONSTRAINT "Mentor_pkey" PRIMARY KEY ("MentorID"),
+    CONSTRAINT "Mentor_fkey" FOREIGN KEY ("MentorID")
+        REFERENCES public."User" ("UserID") MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE IF EXISTS public."Mentor"
+    OWNER to postgres;
+
+
+--5 Table: public.Mentee
+
+-- DROP TABLE IF EXISTS public."Mentee";
+
+CREATE TABLE IF NOT EXISTS public."Mentee"
+(
+    "MenteeID" integer NOT NULL DEFAULT nextval('"Mentee_MenteeID_seq"'::regclass),
+    "LearningHours" double precision NOT NULL,
+    "NumberOfAttandedSessions" integer NOT NULL,
+    CONSTRAINT "Mentee_pkey" PRIMARY KEY ("MenteeID"),
+    CONSTRAINT "Mentee_fkey" FOREIGN KEY ("MenteeID")
+        REFERENCES public."User" ("UserID") MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE IF EXISTS public."Mentee"
+    OWNER to postgres;
+
+
+--6 Table: public.OnlineDonation
+
+-- DROP TABLE IF EXISTS public."OnlineDonation";
+
+CREATE TABLE IF NOT EXISTS public."OnlineDonation"
+(
+    "DonationID" integer NOT NULL DEFAULT nextval('"OnlineDonation_DonationID_seq"'::regclass),
+    "Amount" double precision NOT NULL,
```

Comment:
changed numeric into double precision cause the numeric was giving error when tested